inc/breadcrumbs.php

```diff
@@ -80,11 +80,26 @@ function nightingale_uncanny_breadcrumb_check() {
 function nightingale_breadcrumb() {
 	global $wp_query;
 
-    $show_breadcrumb = get_theme_mod('show_breadcrumb', 'yes');
+    $theme_show_breadcrumb = get_theme_mod('show_breadcrumb', 'yes');
 
-	if ( ! is_home() && $show_breadcrumb == 'yes') {
+	if ( ! is_home() && $theme_show_breadcrumb == 'yes') {
 
-		if ( ! is_front_page() ) {
+        $show_page_breadcrumb = 'yes';
+
+        if(is_singular('page')) {
+            global $post;
+
+            // Get breadcrumb setting, is it set to display or not
+            $show_page_breadcrumb_setting = get_post_meta($post->ID, 'hale_metabox_page_breadcrumb', true);
+
+            if(!empty($show_page_breadcrumb_setting)){
+                $show_page_breadcrumb = $show_page_breadcrumb_setting;
+            }
```

Comment:
Just checking, will the `get_post_meta($post->ID, 'hale_metabox_page_breadcrumb', true);` return a "yes" and also do you need the check if it's empty if you are just changing one variable for another but not setting anything new?

Reply:
it would return a "yes" or "no" if set or if its not set empty. theres 2 variables  $show_page_breadcrumb which defaults as 'yes' unless $show_page_breadcrumb_setting is set.